src/backend/tests/test_crawler_expunger.py

```diff
@@ -198,3 +198,16 @@ def test_probation_revoked_affects_time_eligibility(record_with_revoked_probatio
     expunger_result = expunger.run()
     assert len(expunger_result) == 6
     assert expunger_result[record.cases[0].charges[0].id].date_will_be_eligible == date_class(2020, 11, 9)
+
+
+@pytest.fixture
+def record_with_odd_event_table_contents():
+    return CrawlerFactory.create(
+        record=JohnDoe.SINGLE_CASE_RECORD, cases={"CASEJD1": CaseDetails.CASE_WITH_ODD_EVENT_TABLE_CONTENTS,},
+    )
+
+
+@pytest.mark.skip()
```

Comment:
Why is this new test skipped?

Reply:
Because it fails right now. This doesn't fix the issue. We're doing TDD. Write a failing test (this PR). And then fix it.

Reply:
I'd say it's generally good practice not to skip it if you are doing TDD. Not skipping it has a few benefits. The obvious benefits are that you see it fail first, and then pass when fixed. 

When the failed test is pushed up to a repo as it has been here then the integration test will show that this test is actually failing which is a good thing, because it adds confidence for the reviewers that this test is actually testing the issue that will be fixed. The next commit would then show a passing test further adding confidence for the reviewers that this is a valid fix.

Reply:
Yeah I should have marked this as `pytest.mark.xfail` instead of `skip`.